src/theme/HntToFeeSol.tsx

```diff
@@ -0,0 +1,69 @@
+import useDocusaurusContext from '@docusaurus/useDocusaurusContext'
+import { useConnection, useWallet } from '@solana/wallet-adapter-react'
+import axios from 'axios'
+import React, { useState } from 'react'
+import { useAsyncCallback } from 'react-async-hook'
+import './bufferFill'
+import { Alert, AlertIcon } from './components/Alert'
+import { Button } from './components/Button'
+import { Flex } from './components/Flex'
+import { Wallet, WrapWithAccountProvider } from './components/Wallet'
+import { Transaction } from '@solana/web3.js'
+import { sendAndConfirmWithRetry } from '@helium/spl-utils'
+import { Checkbox } from './components/Checkbox'
+
+// Default styles that can be overridden by your app
+require('@solana/wallet-adapter-react-ui/styles.css')
+
+export const HntToFeeSolImpl = () => {
+  const { siteConfig } = useDocusaurusContext()
+  const url = siteConfig.customFields.HNT_TO_RENT_SERVICE_URL
+  const { publicKey, wallet } = useWallet()
+  const { connection } = useConnection()
+  const { execute, error, loading } = useAsyncCallback(async () => {
+    if (wallet && connection && wallet.adapter && wallet.adapter.connected) {
+      const txRaw = (await axios.post(`${url}/hnt-to-fees`, {
+        wallet: publicKey.toBase58()
+      })).data
+      const tx = Transaction.from(Buffer.from(txRaw))
+      const signed = await wallet.adapter.signTransaction(tx)
```

Comment:
Can you try `signAndSendTransaction` and see if that works?

Reply:
It says that's not a function on adapter